modules/openapi-generator/src/main/resources/JavaSpring/beanValidation.mustache

```diff
@@ -1 +1 @@
-{{#required}}{{^isReadOnly}}@NotNull {{/isReadOnly}}{{/required}}{{#isContainer}}{{^isPrimitiveType}}{{^isEnum}}@Valid {{/isEnum}}{{/isPrimitiveType}}{{/isContainer}}{{^isContainer}}{{^isPrimitiveType}}@Valid {{/isPrimitiveType}}{{/isContainer}}{{>beanValidationCore}}
\ No newline at end of file
+{{^isNullable}}{{^isReadOnly}}@NotNull {{/isReadOnly}}{{/isNullable}}{{#isContainer}}{{^isPrimitiveType}}{{^isEnum}}@Valid {{/isEnum}}{{/isPrimitiveType}}{{/isContainer}}{{^isContainer}}{{^isPrimitiveType}}@Valid {{/isPrimitiveType}}{{/isContainer}}{{>beanValidationCore}}
```

Comment:
does `notNullable` == `required`?

Reply:
no, these are 2 different descriptions for a property.

But I am currently not 100% if `@NotNull JsonNullable<String>` would work. The idea is, that if the value is defined, it must not be `null`. But it is valid that it's `undefined`.

Reply:
according to the documentation in [jackson-databind-nullable](https://github.com/OpenAPITools/jackson-databind-nullable) this should work:

> The module comes with an integrated ValueExtractor that automatically unwraps the contained value of the JsonNullable if used together with javax.validation Bean validation (JSR 380).